eve/main.yml

```diff
@@ -0,0 +1,69 @@
+version: "0.2"
+
+branches:
+  default:
+    stage: "pre-merge"
+
+stages:
+  pre-merge:
+    worker: &pod
+      type: kube_pod
+      path: eve/workers/zenko.yaml
+    steps:
+    - TriggerStages:
+        name: trigger all the tests
+        stage_names:
+        - helm_test_kube_1.9.6
+
+  helm_test_kube_1.9.6:
+    worker:
+      <<: *pod
+      service:
+        requests:
+          version: "1.9.6"
+        namespaces:
+        - "testNamespace"    # <<< the default namespace for that stage
+    steps:
+    - Git: &git_pull
+        name: git pull
+        repourl: "%(prop:git_reference)s"
+        mode: full
+        method: clobber
+        retryFetch: true
+        haltOnFailure: true
+    - ShellCommand: &install_tiller
+        name: install helm (tiller into kubernetes)
+        command: >-
+          helm init --wait
+        env:
+          TILLER_NAMESPACE: '%(prop:testNamespace)s'
+    - ShellCommand:
+        name: Install needed repo
+        command: >
+          helm repo add incubator http://storage.googleapis.com/kubernetes-charts-incubator
+        env:
+          TILLER_NAMESPACE: '%(prop:testNamespace)s'
+    - ShellCommand:
+        name: Retrieve dependency
+        command: >-
+           helm dep build charts/zenko
+        env:
+          TILLER_NAMESPACE: '%(prop:testNamespace)s'
```

Comment:
Might be a good idea to have "haltOnFailure: true" on all the steps since any of these steps will fail if the previous step fail.

Currently looks like it's failing on this step but I believe it's because we have been making many changes to the repo and the branch need to be rebased.